styles/style.css

```diff
@@ -1,176 +1,143 @@
 html {
-  font-size: 12px;
-  font-family: 'Lato','Proxima Nova','Open Sans','Helvetica Neue',Helvetica,Arial,sans-serif;
-  background-color: #transparent!important;
-  letter-spacing: 1.2px;
-  top:100px;
-  right:11px;
-  height:22px;
-  width:100px;
-  font-size:11px;
-  line-height:22px;
-  font-weight:700;
-  letter-spacing:normal;
-  text-align:left;
-  padding: 45px;
-}
-
-{content:"!";
+  font-size:         12px;
+  font-family:       'Lato','Proxima Nova','Open Sans','Helvetica Neue',Helvetica,Arial,sans-serif;
+  background-color:  transparent!important;
+  letter-spacing:    1.2px;
+  top:               100px;
+  right:             11px;
+  height:            22px;
+  width:             100px;
+  font-size:         11px;
+  line-height:       22px;
+  font-weight:       700;
+  letter-spacing:    normal;
+  text-align:        left;
+  padding:           45px;
+  box-sizing:        border-box;
 }
 
 body {
-  width: 1100px;
-  margin: 0 auto;
-  background-color: #white;
-  padding: 0px 20px 20px 20px;
-}
-
-
-/* Nav Bar */
-
-ul {
-  list-style-type: none;
-  margin: 0;
-  padding: 0;
-  overflow: hidden;
   background-color:white;
+  width:             1100px;
+  margin:            0 auto;
+  padding:           0px 20px 20px 20px;
 }
 
-li {
-  float: right;
-  margin: 0px 25px 0px 50px;
-}
-
-li a {
-  display: block;
-  color: gray;
-  text-align: center;
-  text-decoration: none;
-}
-
-li a:hover {
-  color: #111;
-}
+a:active {
+  text-decoration:   none;
+  font-weight:       bold;
+  color:           black!important;
+} 
 
-/* Nav Bar */
+a:link {
+  text-decoration:   none;
+  color:           gray;
+} 
 
+a:visited {
+  text-decoration:   none;
+  font-weight:       normal;
+  color:           gray;
+} 
 
 div {
-  font-size: 13px;
-  text-align: justify;
-  font-weight: 300;
-  color: gray;
-  justify-content: space-between;
-  letter-spacing: 1.15px;
-  line-height: .9;
-  margin-bottom: -15px;
-  padding-bottom: -10px;
-  transform: scale(1, 1.05);
+  font-size:         13px;
+  text-align:        justify;
+  font-weight:       300;
+  color:           gray;
+  justify-content:   space-between;
+  letter-spacing:    1.15px;
+  line-height:       .9;
+  margin-bottom:     -15px;
+  transform:         scale(1, 1.05);
```

Comment:
just a hint this will effect ALL divs, might be more wise to put some of these in a specific class.